scalactic-test/src/test/scala/org/scalactic/AccumulationSpec.scala

```diff
@@ -46,7 +46,7 @@ class AccumulationSpec extends UnitSpec with Accumulation with TypeCheckedTriple
   }
 
   def parseAddress(in: String): List[String] Or One[ErrorMessage] = {
-    val address = in.split(",\\s*")
+    val address = in.split(",\\s*").map(_.nn)
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.